lib/mesh.js

```diff
@@ -274,14 +275,14 @@ Mesh.prototype.__initialize = function (config, callback) {
   _this.log = Logger.createLogger('Mesh');
   _this.log.context = config.name;
 
-  _this.startupProgress("initializing data layer...", 10);
+  _this.startupProgress("initializing data layer...", 56);
```

Comment:
this seems to be the first opportunity to use the current progress tracking mechanism and happens at the start of Mesh.prototype.initialize. I suspect most preceding code is synchronous require() statements. In the meantime the best we can do is #300